test/integration/transactions/escrowFinish.ts

```diff
@@ -0,0 +1,77 @@
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import { EscrowFinish, EscrowCreate, dropsToXrp } from 'xrpl-local'
+
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+import { generateFundedWallet, testTransaction } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('EscrowFinish', function () {
+  this.timeout(TIMEOUT)
+
+  before(suiteClientSetup)
+  beforeEach(_.partial(setupClient, serverUrl))
+  afterEach(teardownClient)
+
+  it('base', async function () {
+    // get the most recent close_time from the standalone container for cancel & finish after.
+    const CLOSE_TIME: number = (
+      await this.client.request({
+        command: 'ledger',
+        ledger_index: 'validated',
+      })
+    ).result.ledger.close_time
+    const wallet1 = await generateFundedWallet(this.client)
+
+    const AMOUNT = 10000
+    const createTx: EscrowCreate = {
+      Account: this.wallet.getClassicAddress(),
+      TransactionType: 'EscrowCreate',
+      Amount: String(AMOUNT),
```

Comment:
For this moving it into a separate variable doesn't make it much more readable (Since you already have the "Amount:" in the object to tell you it's supposed to be an amount. So it would probably be better to just have the raw number there. (In string form if that's the proper type)

Reply:
Can we just make this `Amount: "10000"`?

Reply:
Sure!